src/Twisto/Invoice.php

```diff
@@ -147,6 +151,40 @@ public function returnAll()
         $this->deserialize($data);
     }
 
+
+    /**
+     * Split invoice to new one
+     * @param ItemSplit[] $items
+     * @return Invoice
+     */
+    public function split()
+    {
+        $data = $this->twisto->requestJson('POST', 'invoice/' . urlencode($this->invoice_id) . '/split/', $this->serialize_split());
+        $this->deserialize_split($data);
+        return $invoice;
```

Comment:
`[Thu Apr 12 10:59:45 2018] PHP Notice:  Undefined variable: invoice in .../Twisto/Invoice.php on line 164`